src/test/java/de/uhd/ifi/se/decision/management/jira/eventlistener/consistencycheckeventlistener/TestConsistencyCheckEventListener.java

```diff
@@ -0,0 +1,52 @@
+package de.uhd.ifi.se.decision.management.jira.eventlistener.consistencycheckeventlistener;
+
+import de.uhd.ifi.se.decision.management.jira.eventlistener.Subscriber;
+import de.uhd.ifi.se.decision.management.jira.eventlistener.implementation.ConsistencyCheckEventListenerSingleton;
+import de.uhd.ifi.se.decision.management.jira.rest.EventEmitter;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class TestConsistencyCheckEventListener {
+
+	ConsistencyCheckEventListenerSingleton eventListener;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5586048)